packages/api-explorer/src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Menu, Dropdown as AntDropdown, Icon } from 'antd';
+
+const {Item} = Menu
+
+const styles = {
+  value: {
+    display: 'flex',
+    alignItems: 'center',
+    gap: 8,
+    fontWeight: 'bold',
+  }
+}
+
+const itemMapper = (value) => {
+  return (
+    <Item key={value}>
+      {value}
+    </Item>
+)}
+
+export default function Dropdown ({options, onChange, value}) {
+
+  const onItemChange = ({key}) => onChange(key)
```

Comment:
take these outside the dropdown function

Reply:
I put them inside since they directly depend on onChange and value props. 
Sure, I can take them outside and create a closure, but wouldn't it become too unpractical to read?

Reply:
We can keep it here